recipe/meta.yaml

```diff
@@ -24,7 +24,8 @@ requirements:
     - python
     - numpy
     - python-blosc >=1.4
-    - futures  # [py2k]
+    - enum34   # [py<34]
```

Comment:
@robbmcleod, this is a missing dependency since gh-8. You probably want to add that upstream at https://github.com/em-MRCZ/python-mrcz/blob/v0.5.4/setup.py#L31 too.

Reply:
Done, thanks.

Reply:
Perfect, thanks!